fly.toml

```diff
@@ -0,0 +1,46 @@
+# fly.toml file generated for audio-discrimination-croudsource-dev on 2023-02-15T10:46:48+03:00
+
+app = "audio-discrimination-croudsource-dev"
```

Comment:
I don't love this because if I have several devs, all of them will be using the app name that I want.

There should be a setup script where the dev gives their name and it creates a config for them that sets up their app name, database name, etc.

Consider maybe making a fly.toml.tmpl (template) which is in git, a setup script that creates fly.toml, and that is part of .gitignore so people don't check in their secrets